README.md

```diff
@@ -1,207 +1,145 @@
-# HOT Shared UI
-
-<!-- markdownlint-disable -->
-<p align="center">
```

Comment:
I preferred this centralised layout, as it aligns with READMEs in our other tools too

Reply:
Yes? it's not very clean ..

Reply:
Clean based on what metric?

Sure it's HTML in Markdown which isn't ideal, but it makes for a nicer looking README file so it's worth it.

So many other projects use this approach 😅

Reply:
I mean, I don't think the design and content organization is clean, using HTML is ok. IMO it should be easier to understand, with more consistent alignment and without excessive amount of information.